GlotPress/gp-translation-helpers
# Allow to add a comment and approve or fuzzy the translation

## Problem

When you add some comments in a single action, if you approve or fuzzy the translation(s), the comment is not stored in the database, and we don't send any notifications.

See https://github.com/GlotPress/gp-translation-helpers/issues/91.

![image](https://user-images.githubusercontent.com/1667814/181226618-9a308464-364a-4345-83c5-262a1ed99cf9.png)

![image](https://user-images.githubusercontent.com/1667814/181226983-48ff996b-a83b-427b-b2be-0df5e3f50551.png)

We don't need to update the email template:

![image](https://user-images.githubusercontent.com/1667814/181227151-aeac10e3-d184-46bc-8037-0d2433025a64.png)



## Solution

We need to:
* Store the comment when the translation is set to approved or fuzzy.
* Update the template discussion, removing some references to "reject" or "rejection", because the comment can be related with other translation state.
* Send the notification to the translator.

This PR doesn't update the bulk rejection, so the users only can make comments in the bulk rejection, not in the bulk approve or bulk fuzzy.

Updates in the discussion template:

![image](https://user-images.githubusercontent.com/1667814/181284108-f760afb1-5510-4b28-850a-52d07fceb189.png)



## Testing instructions

* Make some comment in a pending translation.
* Set as approved or fuzzy.
* Go to the discussion page to see the comment.
* Check that the translator has received the email notification.
* Test that the bulk rejection is working fine.
